contracts/contract_transactions.md

```diff
@@ -0,0 +1,68 @@
+[back](./contracts.md)
+## Contract Transactions
+
+Contract transactions are of three types:
+- Create
+- Call
+- Disable -- To be decided.
+
+A contract is also a normal account with a balance.
+
+
+### Create Contract Transaction
+
+Anyone can create a new contract by submitting a create contract transaction.
+
+The transaction contains:
```

Comment:
Shall this contain also a (even) deposit? See contracts/ring.md "killing a contract"

Reply:
Yes. Fixed.